components/compliance-service/inspec/cli.go

```diff
@@ -78,22 +94,41 @@ func Scan(paths []string, target *TargetConfig, timeout time.Duration, env map[s
 	target.ResultIncludeBacktrace = false
 	target.ResultMessageLimit = ResultMessageLimit
 
-	env["HOME"] = os.Getenv("HOME")
+	//env["HOME"] = os.Getenv("HOME")
 
-	args := append([]string{binName, "exec"}, paths...)
+	//args := append([]string{binName, "exec"}, paths...)
 
 	// write the inputs to a file to be passed to inspec during command execution
+	tmpDirPath := fmt.Sprintf("/tmp/inspec-upload-%v", makeTimestamp())
+	args, err := getFirejailArgsaAndOutputFileForExec(fireJailExecProfilePath, paths, tmpDirPath)
+	if err != nil {
+		return nil, nil, NewInspecError(INVALID_PARAM, err.Error())
+	}
+
+	tmpKeys := copyKeyFilesIntoTmpDirectory(tmpDirPath, target.KeyFiles)
+	newTarget := target
+	newTarget.KeyFiles = tmpKeys
+	stdoutFile, erroutFile, shellFile := shellscriptAndResponse(exec_command, tmpDirPath)
+	commandArgs := args
+	commandArgs = append(commandArgs, []string{"/bin/sh", shellFile}...)
+	commandArgs = append(commandArgs, paths...)
```

Comment:
Can we combine line no 112 to 114 in 1 line.

Reply:
We can, but both of the lines are doing the same thing. It's just making the code in one line. This is making it more readable. @kalroy thoughts?